cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go

```diff
@@ -320,14 +317,3 @@ type HostPathMount struct {
 	// PathType is the type of the HostPath.
 	PathType v1.HostPathType `json:"pathType,omitempty"`
 }
-
-// AuditPolicyConfiguration holds the options for configuring the api server audit policy.
-type AuditPolicyConfiguration struct {
```

Comment:
From @fabriziopandini: We shouldn't remove from the existing v1alpha3 API, instead error if the field was set in v1alpha3 when converting to v1beta1

Reply:
Also add a conversion_test.go unit test to error if it was set in v1alpha3